install/kubernetes/helm/istio/charts/sidecar-injector/templates/mutatingwebhook.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Capabilities.APIVersions.Has "admissionregistration.k8s.io/v1beta1" -}}
+{{- if or (.Capabilities.APIVersions.Has "admissionregistration.k8s.io/v1beta1") (and (not .Release.IsInstall) .Values.useAdmissionRegistration) -}}
```

Comment:
How will this work ? It'll be created with the invalid CA bundle ?

Reply:
I was actually going to put exactly that in another PR (I'm overriding it locally for now) but I can include it here.